---
title: "Reconstructing rodent contact networks to understand potential routes of Lassa mammarenavirus transmission."
abstract: |
  Lassa fever, caused by Lassa mammarenavirus (LASV), is a zoonotic infectious disease endemic in several West African countries. Human infection is caused by spillover from rodent host species, the primary reservoir is proposed to be Mastomys natalensis, a commensal rodent species. In addition to the primary reservoir a further 11 rodent species have been found to be acutely infected or have evidence of prior infection. The contribution of these other species to viral transmission among rodent communities is not understood. Here, we use a rodent ecology study conducted in a Lassa fever endemic region to investigate prevalence of antibodies against LASV among rodent communities in different landuse types. We reconstruct rodent contact networks in these settings from the detection of 601 rodent individuals from 37,982 trap-nights. We report a seroprevalence of 3.3% among these rodent communities with antibodies detected in 6 rodent species. We found that rodent communities were more connected within agricultural landuse settings than within villages or forest sampling sites. This, combined with our finding of increased odds of intraspecific contact among Mastomys natalensis within agricultural compared to village landuse suggests that LASV transmission may occur at greater rates within the more species rich agricultural settings than within villages. Our findings suggest that to better understand transmission dynamics of LASV within endemic settings sampling of the entire rodent community is required. Expanding rodent trapping to both village and agricultural landuse settings may elucidate the rodent species that are important for the maintenance of viral populations and the risk of zoonotic spillover into human populations.
author: 
  - name: "David Simons"
    affiliation: The Royal Veterinary College
categories: [Rodent, Ecology, Zoonosis, Lassa Fever, In Preparation, Network, Transmission]
image: files/rodent_network/preview_image.png
bibliography: rodent_network.bib
date: 2023-05-01
citation: true
execute:
  echo: false
  warning: false
---
```{r}
library(dplyr)
library(readr)
library(here)
library(flextable)
```

**This work is currently in preparation with additional data expected**

## Motivation

This study was designed to explore the contacts among individual rodents trapped in our ongoing rodent trapping study in a Lassa fever endecmic region of Sierra Leone. While there is some understanding of where different individuals of rodent species are located across landuse gradients in the endemic region there has been little research on the probability of contact among individuals of these species and what this may mean for viral transmission in these settings. Direct and indirect contact between infected and susceptible individual rodents will modify how viral pathogens are transmitted in these settings and there may be different rates of contacts between individuals of different species or in different landuse types that could be important for understanding viral transmission.

We used data from our repeated, systematic, rodent trapping study in the Eastern province of Sierra Leone to investigate the following questions. First, what are the potential contact networks of individuals of different species. Second, do these contact rates vary by landuse type. Third, do we observe differential inter- and intra-specific contact rates among the primary rodent reservoir of LASV (*Mastomys natalensis*). To achieve this we reconstructed contact networks of rodent communities using geolocated trap data for each of our study visits. We further report the prevalence of antibodies to LASV among rodents in our study but do not currently combine this data with our contact networks due to few positive samples.

## Method

Rodent sampling is described in further detail [elsewhere](/rodent_trapping.html).

### Study area

Rodent trapping surveys were conducted between October 2020-February 2023 within and around four village study sites (Baiama; latitude = 7.8375, longitude = -11.2683, Lalehun; latitude = 8.1973, longitude = -11.0803, Lambayama; latitude = 7.8505, longitude = -11.1969, and Seilama; latitude = 8.1224, longitude = -11.1936) in the Lassa fever endemic zone of the Eastern Province of Sierra Leone. Surveys were conducted within trapping grids along a landuse gradient of anthropogenic disturbance comprising, forest, agriculture (including fallow and currently in-use areas), and villages (within and outside of permanent structures). Trapping survey sessions within each village occurred four times annually with two trapping surveys in each of the rainy and dry seasons (May to November and December to April, respectively), producing a total of 9 trapping sessions over the study period.

Village study sites and trapping grids within the village study sites were selected to be representative of land use in the Eastern Province of Sierra Leone and based on accessibility to the sites during all seasons alongside acceptability of the study protocol to the village study site communities. [Supplementary Material 1](files/rodent_network/Supplementary-Material-1.pdf) contains detailed information about the trapping process. Briefly, at each trapping grid 49 Sherman traps (7.62cm x 8.89cm x 22.86cm) (H.B. Sherman Traps, Tallahasee, USA), were placed in a 7 trap by 7 trap grid, traps were placed 10 metres apart in a grid conforming to the local landscape (median trapping grid area = 4,813m2). For traps placed within permanent structures trap placement deviated from the grid structure. Permanent structures were selected randomly at each visit from a grid projected over the village area, with four traps placed within each structure. The location of each individual trap within trapping grids was geolocated. Traps were baited with a locally produced mixture of oats, palm oil and dried fish. Each morning the traps were checked and closed for the day prior to re-baiting during the evening. Each trapping survey session consisted of four consecutive trap-nights (TN) at each trapping grid within the village study site.

Trapped rodents were associated with the coordinates of the trap they were detected in. The `sf` package in the R statistical computing language (R version 4.1.2) was used for geospatial manipulation and analysis [@pebesma_simple_2018; @r_core_team_r_2021]. All rodent handling was performed by trained researchers, rodents were sedated with halothane and euthanised prior to obtaining morphological measurements and samples of blood and tissue following published guidance [@fichet-calvet_chapter_2014]. The study protocol was approved by the Clinical Research Ethical Review Board and Animal Welfare Ethical Review Board of the Royal Veterinary College, United Kingdom (URN: 2019 1949-3), and Njala University, Sierra Leone. Carcasses were destroyed through incineration to eliminate the risk of onward pathogen transmission.

### Species classification

Taxonomic identification was performed in the field based on external characteristics using a taxonomic key, including external morphological measurements and characteristics, developed from Kingdon and Monadjem [@happold_mammals_2013; @monadjem_rodents_2015]. Morphological identification alone is unable to distinguish some small-mammal species within the study area at species level. Therefore, molecular identification was performed on whole blood, tissue or dried blood spots. Samples were stored at -20°C until processing, genomic DNA was extracted using QIAGEN DNAeasy kits as per the manufacturers instructions ([Supplementary Material 2](/files/rodent_network/Supplementary-Material-2.pdf)) [@qiagen_dneasy_2023]. DNA extracts were amplified using platinum *Taq* polymerase (Invitrogen) and cytochrome B primers [@bangura_lassa_2021]. DNA amplification was assessed through gel electrophoreisis with successful amplification products undergoing Sanger sequencing. Attribution of obtained sequences to rodent species was through the BLAST programme comparing NCBI species records for rodent cytochrome B to our sample sequences [@altschul_basic_1990].

### Lassa mammarenavirus serology

The BLACKBOX® LASV IgG ELISA Kit developed by the Diagnostics Development Laboratory hosted at the Bernhard Nocht Institute for Tropical Medicine and validated for rodent samples was used to determine serological status of trapped rodents [@gabriel_development_2018; @soubrier_detection_2022]. The full protocol is available as Supplementary Material 1. Briefly, 1 µL of whole blood was inactivated by mixing with the provided sample dilution buffer (1:50). Where whole blood was unavailable, blood was extracted from dried blood spots stored on filter paper by incubating with phosphate-buffered saline containing 0.08% Sodium Azide and 0.05% Tween-20. Samples alongside negative and positive controls were incubated on the provided ELISA plates for 24 hours at 4-8 °C in a wet chamber. Following incubation, the plates were washed and incubated for a further hour with 1:10,000 diluted HRP-labelled streptavidin. A final wash was performed prior to the addition of 100µL of 3,3’,5,5’-Tetramethylbenzidine (TMB) substrate to wells, with incubation for 10 min. The colorimetric reaction was stopped by adding 100µL of a stop solution.

In a deviation from the kit protocol the optical density (OD) at 450nm and 630nm was measured (as opposed to 450nm and 620nm). The index value was produced from the OD difference (OD~450~-OD~630~) divided by the cut-off values (the mean values of the negative controls + 0.150). Samples were considered positive with index values greater or equal to 1.1, negative results less than or equal to 0.9, and inconclusive results when the index value lay between 0.9 and 1.1. Inconclusive results were repeated as advised by the kit manufacturers.

### Preparing rodent contact networks

Rodent contact networks were reconstructed from rodent trapping data. Capture-mark-recapture (CMR) methods have previously been used to identify space-sharing by individuals [@carslake_spacetime_2005; @clay_contact_2009; @wanelik_new_2022]. Within our system a CMR design was not possible due to the risk of releasing an infected rodent back into human communities. We therefore consider that rodents experience direct or indirect contacts with other individuals through detections at trapping locations co-located in time and space [@perkins_comparison_2009]. We assumed these potential contacts were sufficient to transmit LASV if they were trapped within a buffer zone of 30m radius (2,828 m2) from the location of the trap during the same 4 trap night session. A 30m radius was selected to encompass the potential home range of an individual, a strong assumption underlying this approach is that an individual was trapped at the center of their home range [@wanelik_new_2022]. This buffer was applied to all species, further assuming that each species shared the same size home range. We assessed the appropriateness of the choice of 30m as our buffer radius and this second assumption using the `HomeRange` R package (version 1.0.2) [@broekman_homerange_2023]. Four of our identified species, and a further four species from genera of our trapped species are included in this dataset. A 30m buffer radius contains the entirety of detected home ranges of individuals of M. natalensis our primary species of interest, and greater than 50% of the individuals of *Lemnisomys striatus*, *Mus musculus* and *Rattus rattus* (81%, 92% and 52% respectively) (Supplementary Figure 1.). Sensitivity analyses were performed using buffer areas of 15m and 50m.

Networks were constructed from observed individuals (nodes) and the presence or absence of contacts between them (edges). Data were aggregated at landuse type and sampling visit producing a potential 28 distinct networks from 131 trapping grid, village and visit combinations. Only 25 networks were produced as there were no observed individuals in three of the potential networks produced from forest landuse settings.

Within our trapping grids only a subset of all active rodents are detected in traps. Prior analysis of our study system suggests a probability of detection at each trap of less than 10% for 4 trap nights if the species is present in the trapping grid. Including unobserved individuals, and therefore unobserved contacts between these individuals aids interpretation of network models, by providing a measure of the total population size that our analytic sample is derived from. To estimate the abundance of individuals of each species within a trapping grid we modelled abundance (total population size) from repeated count data using an N-mixture model implemented in the `unmarked` R package (version 1.2.5) [@royle_n-mixture_2004; @fiske_unmarked_2011]. The latent abundance distribution can be modelled as a Poisson, negative binomial or zero-inflated Poisson random variable. The abundance distribution was modelled with the number of trap nights and season as replicate dependent detection covariates in addition to location (whether a site was based in a rural or peri-urban setting) and landuse type (forest, agriculture or village) as occurrence covariates. To select the most appropriate model for each species, the AIC of each of the Poisson, negative binomial or zero-inflated Poisson abundance distribution models were compared, with the best fitting model used to derive the estimated abundance. The median estimated abundance from the produced distribution at a trapping grid was then used to generate the unobserved individuals within each network aggregated to landuse type (Supplementary Figures 2A-F). The number of observed individuals was then subtracted from the abundance to derive the number of unobserved individuals of each species. These unobserved individuals were explicitly set to have missing (i.e., unobserved) edge values.

Finally, the constructed adjacency matrices were converted to networks using the `network` R package (version 1.13.0.1) [@butts_network_2008]. We describe these inferred contact networks at landuse and visit level using network metrics including the number of nodes, the number of edges, the number of unobserved nodes and edges, median node degree and network density. Species level descriptions of the number of contacts by species and landuse are reported. Graphical representations of the inferred networks are shown in Supplementary Figure 3A-W.

### What is the association of landuse- and species-level heterogeneity on rodent contact networks?

To investigate whether landuse and species are associated with the probability of a contact between two individuals we model these contacts as Exponential-Family Random Graphs (ERGM) [@hunter_ergm_2008]. ERGMs implement maximum likelihood estimates to produce an Odds Ratio for the probability of an edge forming following the addition of a new node into a network based on network properties and nodal attributes. ERGMs were produced for each of our inferred contact networks to compare the probabilities of edges forming based on rodent characteristics (i.e., species). The general model term follows Equation 1 (@eq-1).

$$
\log({\exp(\theta'g(y))}) = \theta_1g_1(y) + \theta_2g_2(y)+ ... + \theta_pg_p(y)
$$ {#eq-1}

Here $p$ is the number of terms in the model, the coefficients $\theta$ represent the size and direction of the effects of the covariates $g(y)$ on the overall probability of an edge forming in the network. At the edge level the expression for the probability of the entire graph can be re-expressed as the conditional log-odds of a single edge between two nodes (a contact between two rodents) as in Equation 2 (@eq-2):

$$
\operatorname{logit}P{(Y_{ij}=1|y^{c}_{ij})=\theta'\delta(y_{ij})}
$$ {#eq-2}

Here $Y_{ih}$ is the random variable for the state of the node pair $ij$ and $y^{c}_{i,j}$ signifies all dyads in the network other than $y_{i,j}$. $\theta'$ is the coefficient for the change production of an edge between the two nodes conditional on all other dyads remaining the same ($\delta(y_{ij}$). 

ERGMs are implemented using the `ergm` package (version 4.3.2) in R [@handcock_ergm_2022]. Three terms were included in the final ERGM to model the probability of the formation of ties (Equation 3. (@eq-3)). The first term (edges), describes the density of the network, and is the probability of a tie being observed in the network. The second term (species), is the conditional probability of a tie forming conditional on the species of the new node. The third term (species homophily), is the conditional probability of a tie forming accounting for intraspecific tie formation among rodent individuals (i.e., the conditional probability of two individuals of the same species forming a tie). To reduce linear dependency of the nodal terms and due to data sparsity within our inferred networks all non-*M. natalensis* are grouped as "Other species" through the `levels` term of the nodal covariates.

$$
\text{Probability of tie}= \text{edges} + \text{species} + \text{species homophily}
$$ {#eq-3}

Models with unstable estimates for the species homophily term were not included in the random-effects meta-analysis. No contact networks from forest landuse contributed to meta-analysis as no *M. natalensis* were detected in these settings. Five models from agricultural settings and 6 from village settings were included in meta-analysis. Random effects models were specified using the `metafor` package (version 3.8-1) in R [@viechtbauer_conducting_2010]. Effect sizes and standard errors for the three model terms were extracted. Weights for each network in the meta-analysis were assigned using inverse-variance weights. Two sensitivity analysis were performed first, by specifying a multi-level structure to the random-effects meta-analysis and second, by performing leave one-out meta-analysis [@cheung_guide_2019]. Forest plots were produced to visualise the summary Odds Ratio of the probability of a tie for each model term stratified by landuse type.   


## Results 

### Antibodies against LASV are detectable in multiple rodent species within an endemic region

601 individual rodents were trapped from 37,982 trap-nights (TN). 13 species were identified from molecular classification, the majority of which were identified as *M. natalensis* (N = 102, 17%), *Praomys rostratus* (N = 88, 14.6%) and *Mus musculus* (N = 73, 12.1%) (Table 1.). Antibodies to LASV were identified in 20 rodents (20/601, 3.3%) from 6 species, including *M. natalensis* (7/20, 35%), 5 *Crocidura spp.* (5/20, 25%), 3 *L. sikapusi* (3/20, 15%) and 3 *Mus setulosus* (3/20, 15%) (Table 1.). The highest proportion of positivity was observed in *Malacomys edwardsi* (1/11, 9%), *Mus setulosus* (3/38, 7.9%) and *M. natalensis* (7/102, 6.8%).

```{r, fig.cap = "Table 1: The number of individuals detected and antibodies to *Lassa mammarenavirus* among those individuals.", echo=FALSE, message=FALSE, warning=FALSE}

read_rds(here("lassa", "files", "rodent_network", "Table_1.rds")) %>%
  autofit()

```

Rodents with antibodies to LASV were detected in three of the study villages, Lalehun (N = 11, 55%), Seilama (N = 8, 40%) and Baiama (N = 1, 5%). Lalehun had the highest percentage of antibody positive rodents (11/146, 7.5%), followed by Seilama (8/247, 3.2%) and Baiama (1/96, 1%), no positive rodents were detected in the most urbanised village Lambayama. Antibody positive rodents were detected in all landuse types, most positive rodents were trapped in agricultural landuse (13/20, 65%), followed by village (6/20, 30%) and forest (1/20, 5%) settings. The proportion of antibody positive individuals among all rodents trapped were similar across forest (1/40, 2.5%), agricultural (13/339, 3.8%) and village (6/222, 2.7%) landuse types. Antibody positive rodents were detected during all sampling visits, the proportion of rodents testing positive were similar between the dry (11/364, 3%) and rainy (9/237, 3.8%) seasons.

### Rodent contact rates, while heterogeneous, are similar across landuse types

Networks produced from rodents trapped in agricultural landuse settings contained the highest species richness (median = 10, IQR = 1), followed by village (median = 8, IQR = 1) and forest (median = 6, IQR = 0.5). More individuals (nodes) were observed and predicted in agricultural landuse (median = 988, IQR = 118) than village (median = 447, IQR = 317) and forest (median = 163, IQR = 32.5) (Figure 1A.). The number of observed contacts (edges) followed a similar pattern with a greater number of edges observed in agricultural settings (median = 38, IQR = 47) followed by village (median = 26, IQR = 19) and forest landuse (median = 3, IQR = 16.5) (Figure 1B.). Network density, was similar in village and agricultural settings (median = 0.06 and 0.09 respectively) but higher in forest settings (median = 0.44) where fewer individuals were observed. 

```{r, fig.cap =  "Figure 1: A) The number of nodes in each of the 25 networks produced from each sampling visit in different landuse types. The number of observed rodents (nodes) is shown in the solid colour, the number of undetected nodes in these networks are shown in pale colours. B) The number of edges formed between observed nodes in each of the 25 networks."}

knitr::include_graphics(here("lassa", "files", "rodent_network", "Figure_1.png"))

```

The median degree (number of edges) of a node were similar across all landuse types (Figure 2A.). Within forest landuse types no rodents had a degree greater than 6, the highest degree was 19 and 17 in agricultural and village landuse respectively. The median number of contacts did not importantly differ by species (Figure 2B.). Within species there was substantial variability in degree between individual rodents, for example, the median degree of *Praomys rostratus* in agricultural landuse was 3, although 3 individuals had a degree of greater than 15.

```{r, fig.cap =  "Figure 2: A) The degree of each observed rodent within the produced networks grouped by landuse type. B) The distribution of degree within the networks grouped by the species of the observed individual and the landuse type in which they were detected."}

knitr::include_graphics(here("lassa", "files", "rodent_network","Figure_2.png"))

```

We did not observe any important difference in the median number of contacts for species that were found to be LASV antibody positive. 

### Intraspecific contacts dominated observed contacts

Individual species had high variability in the number of species they had contact with, for example *M. natalensis*, *P. rostratus* and *R. rattus* had contacts with 9 other rodent species while *M. musculus* had 3 (Figure 3.). There was a general trend that the species with more individuals observed had greater number of contacts with other species (*r(11)* = 0.78, *p* < 0.005). *M. musculus* is an important outlier to this trend, it was the fourth most observed species but had few observed contacts with other species. Intraspecific contacts dominated the edges between individual rodents of most species, across all landuse types, particularly among the most commonly observed species. *Mastomys natalensis* was found to have contacts with 8 other species in agricultural landuse and 5 in village landuse settings, in agricultural settings most contacts were observed to be intra-specific while in villages interspecific contacts with *R. rattus* were more commonly observed than both intraspecific contacts and contacts with another commensal species, *P. rostratus*. 

```{r, fig.cap =  "Figure 3: The number of contacts between individuals of the different rodent and small mammal species detected across the three landuse types, Forest (A), Agriculture (B) and Village (C). Pale yellow cells represent no observed contacts between members of these species. Darker colours indicate increasing numbers of observed contacts between species'."}

knitr::include_graphics(here("lassa", "files", "rodent_network", "Figure_3.png"))

```

All six species containing individuals found to be LASV antibody positive in this study were found to have contact with *M. natalensis*, the primary reservoir of LASV, either in agricultural or village landuse settings. 

### The association of landuse- and species-level heterogeneity on rodent contact networks

Limiting our analysis of the odds of a contact being observed to the primary reservoir species of LASV, *M. natalensis*, resulted in 11 of the constructed networks being suitable for random effects meta-analysis. Four agricultural and 6 village landuse networks were incldued in meta-analysis (Figure 3.). No forest landuse networks were suitable for analysis as no *M. natalensis* were detected in these settings. The odds of a contact being observed in these networks were generally low with similar odds across both agricultural (Odds Ratio = 0.08, 95% Confidence Interval = 0.04-0.13, *p* < 0.001) and village landuse (OR = 0.1, 95% C.I. = 0.06-0.16, *p* < 0.001). *Mastomys natalensis* had a statistically significant reduced odds of forming a contact with another rodent in agricultural settings compared with other species in these communities (OR = 0.48, 95% C.I. = 0.26-0.9, *p* = 0.02). In village landuse types the odds of contact for *M. natalensis* was similar to other species (OR = 0.69, 95% C.I. = 0.4-1.19, *p* = 0.18). *Mastomys natalensis* had a statistically significant increased odds of forming an intraspecific contact compared to interspecific contacts in agricultural settings (OR = 6.11, 95% C.I. = 2.7-13.8, *p* < 0.001) but not in village landuse types (OR = 2, 95% C.I. = 0.7-5.64, *p* = 0.19). These meta-analysis showed a variable degree of heterogeneity. Substantial heterogeneity was found in the odds of a contact being observed in the network (I^2^~agriculture~ = 94% and  I^2^~village~ = 82%) and a contact being observed for *M. natalensis* compared to other species (I^2^~agriculture~ = 70% and I^2^~village~ = 57%) for both landuse types. Heterogeneity in the analysis of the odds of a contact between individuals of *M. natalensis* was substantially lower (I^2^~agriculture~ = 9% and I^2^~village~ = 38%) in both landuse types. In sensitivity analysis the directions of these odds ratios did not vary in the first sensitivity analysis of altering the radius in which a contact was defined or in the second sensitivity analyses of leave one out random effects meta-analysis suggesting the results are robust to these two assumptions.

```{r, fig.cap =  "Figure 4: Random effects meta-analysis of ERGM network models investigating the odds of a contact being observed for *M natalensis*. A) The odds ratio of a contact being observed for *M. natalensis* in Agricultural or Village landuse types. B) The odds ratio of a contact being observed between *M. natalensis* and an individual of a different rodent species. C) The odds ratio of a contact being observed between *M. natalensis* and another *M. natalensis*."}

knitr::include_graphics(here("lassa", "files", "rodent_network", "Figure_4.png"))

```

## Discussion

In the current study we have described seropositivity to LASV within the Eastern province of Sierra Leone, we found low numbers of individuals with evidence of prior infection, from 6 rodent species. The majority of individuals with evidence of prior infection were individuals of the primary reservoir of LASV *M. natalensis*. We found that constructed contact networks were generally larger in both agricultural and village landuse settings, with similar contact rates, compared to forest landuse types. Finally, focussing on the primary LASV reservoir we identified that *M. natalensis* were less likely to form contacts but were more likely to come into contact with members of the same species in agricultural landuse types. The same pattern was not observed in village landuse types where individuals were as likely as other species to form contacts and were not found to have a higher odds of forming intraspecific contacts. This is potentially important for the dynamics of LASV transmission in these settings and suggests that pathogen transmission among the rodent community may occur at greater rates in agricultural landuse than village settings.


## Supplementary material

### [Supplementary Material 1](/files/rodent_network/Supplementary-Material-1.pdf)

### [Supplementary Material 2](/files/rodent_network/Supplementary-Material-2.pdf)

### Supplementary Figure 1

```{r S.fig.1, fig.cap = "Supplementary Figure 1: Empirical Cumulative Density Function of the home range radius of rodent and shrew species with data available in the HomeRange dataset. Species that match detected genera in our study include two shrew species *Crocidura leucodon* and *Crocidura shantungensis* and two rodent species *Hylomyscus stella* and *Praomys tullbergi*. Four species matches to rodent species detected in our study were also included *Lemniscomys striatus*, *Mastomys natalensis*, *Mus musculus* and *Rattus rattus*. Only *Lemniscomys striatus* and *Mastomys natalensis* contain data from Africa (Uganda and Tanzania respectively). The dashed line represents the 30m range radius used for the primary analysis in the current study."}

knitr::include_graphics(here("lassa", "files", "rodent_network", "Supplementary_Figure_1.png"))

```

### Supplementary Figure 2A-F

```{r S.fig.2A, fig.cap = "Supplementary Figure 2A: Estimated abundance at each sampling site for *Mastomys natalensis*. The dashed line and number is the median abundance used to infer the population size at this study site."}

knitr::include_graphics(here("lassa", "files", "rodent_network", "Supplementary_Figure_2A.png"))

```


```{r S.fig.2B, fig.cap = "Supplementary Figure 2B: Estimated abundance at each sampling site for *Mus musculus*. The dashed line and number is the median abundance used to infer the population size at this study site."}

knitr::include_graphics(here("lassa", "files", "rodent_network", "Supplementary_Figure_2B.png"))

```

```{r S.fig.2C, fig.cap = "Supplementary Figure 2C: Estimated abundance at each sampling site for *Rattus rattus*. The dashed line and number is the median abundance used to infer the population size at this study site."}

knitr::include_graphics(here("lassa", "files", "rodent_network", "Supplementary_Figure_2C.png"))

```

```{r S.fig.2D, fig.cap = "Supplementary Figure 2D: Estimated abundance at each sampling site for *Crocidura spp*. The dashed line and number is the median abundance used to infer the population size at this study site."}

knitr::include_graphics(here("lassa", "files", "rodent_network", "Supplementary_Figure_2D.png"))

```

```{r S.fig.2E, fig.cap = "Supplementary Figure 2E: Estimated abundance at each sampling site for *Praomys spp*. The dashed line and number is the median abundance used to infer the population size at this study site."}

knitr::include_graphics(here("lassa", "files", "rodent_network", "Supplementary_Figure_2E.png"))

```

```{r S.fig.2F, fig.cap = "Supplementary Figure 2F: Estimated abundance at each sampling site for all other species (species with fewer than 20 observations). The dashed line and number is the median abundance used to infer the population size at this study site."}

knitr::include_graphics(here("lassa", "files", "rodent_network", "Supplementary_Figure_2F.png"))

```

### Supplementary Figure 3A-W

![Supplementary Figure 3A](files/rodent_network/Supplementary_Figure_3A.png)

![Supplementary Figure 3B](files/rodent_network/Supplementary_Figure_3B.png)

![Supplementary Figure 3C](files/rodent_network/Supplementary_Figure_3C.png)

![Supplementary Figure 3D](files/rodent_network/Supplementary_Figure_3D.png)

![Supplementary Figure 3E](files/rodent_network/Supplementary_Figure_3E.png)

![Supplementary Figure 3E](files/rodent_network/Supplementary_Figure_3E.png)

![Supplementary Figure 3F](files/rodent_network/Supplementary_Figure_3F.png)

![Supplementary Figure 3G](files/rodent_network/Supplementary_Figure_3G.png)

![Supplementary Figure 3H](files/rodent_network/Supplementary_Figure_3H.png)

![Supplementary Figure 3I](files/rodent_network/Supplementary_Figure_3I.png)

![Supplementary Figure 3J](files/rodent_network/Supplementary_Figure_3J.png)

![Supplementary Figure 3K](files/rodent_network/Supplementary_Figure_3K.png)

![Supplementary Figure 3L](files/rodent_network/Supplementary_Figure_3L.png)

![Supplementary Figure 3M](files/rodent_network/Supplementary_Figure_3M.png)

![Supplementary Figure 3N](files/rodent_network/Supplementary_Figure_3N.png)

![Supplementary Figure 3O](files/rodent_network/Supplementary_Figure_3O.png)

![Supplementary Figure 3P](files/rodent_network/Supplementary_Figure_3P.png)

![Supplementary Figure 3Q](files/rodent_network/Supplementary_Figure_3Q.png)

![Supplementary Figure 3R](files/rodent_network/Supplementary_Figure_3R.png)

![Supplementary Figure 3S](files/rodent_network/Supplementary_Figure_3S.png)

![Supplementary Figure 3T](files/rodent_network/Supplementary_Figure_3T.png)

![Supplementary Figure 3U](files/rodent_network/Supplementary_Figure_3U.png)

![Supplementary Figure 3V](files/rodent_network/Supplementary_Figure_3V.png)

![Supplementary Figure 3W](files/rodent_network/Supplementary_Figure_3W.png)

## Data availability and draft manuscript

Data are available in the project's GitHub [repository](https://github.com/DidDrog11/chapter_4/tree/main/data). The most recent version of the draft currently being shared with co-authors is [V2](https://github.com/DidDrog11/chapter_4/blob/main/report/chapter_4.docx).
